src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -60,20 +70,51 @@ class pt3(Macro):
     """Macro with a list of numbers as parameter: the type is a sequence of
     parameter types which is repeated. In this case it is a repetition of a 
     float so only one parameter is defined.
-    By default the repetition as a semantics of 'at least one'"""
+    By default the repetition as a semantics of 'at least one'
+    Usages, ex.:
+    pt3 [[1][34][15]]
+    pt3 [1 34 15]
+    pt3 1 34 15
+    """
 
     param_def = [
        [ 'numb_list', [ [ 'pos', Type.Float, None, 'value'] ], None, 'List of values'],
     ]
     
     def run(self, *args, **kwargs):
         pass
+    
+class pt3d(Macro):
+    """Macro with a list of numbers as parameter: the type is a sequence of
+    parameter types which is repeated. In this case it is a repetition of a 
+    float so only one parameter is defined. The parameter has a default value.
+    By default the repetition as a semantics of 'at least one'
+    Usages, ex.:
+    pt3d [[1][34][15]]
+    pt3d [1 34 15]
+    pt3d 1 34 15
+    Usages taken the default value, ex.:
+    pt3d [[1] [] [15]]
+    pt3d [1 [] 15]
+    """
+
+    param_def = [
+       [ 'numb_list', [ [ 'pos', Type.Float, 21, 'value'] ], None, 'List of values'],
+    ]
+    
+    def run(self, *args, **kwargs):
+        pass
 
 class pt4(Macro):
     """Macro with a list of motors as parameter: the type is a sequence of
     parameter types which is repeated. In this case it is a repetition of a 
     motor so only one parameter is defined.
-    By default the repetition as a semantics of 'at least one'"""
+    By default the repetition as a semantics of 'at least one'.
+    Usages, ex.:
+    pt3 [[mot1][mot2][mot3]]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.